dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -406,9 +406,10 @@ internal string ExpandFromVariables(ContextVariables variables, string input)
 
```

Comment:
Should we add a `.Distinct` here to remove duplicates? (The `.Replace(...)` below will replace all instances, so it would save an extra lookup, and the subsequent Replace calls for the same varName would not find the varName in the string anymore anyway).